docs/scripts/reportBrokenLinks.js

```diff
@@ -0,0 +1,209 @@
+const path = require('path');
+const fse = require('fs-extra');
+const { createRender } = require('@mui/markdown');
+const { marked } = require('marked');
+
+const UNSUPPORTED_PATHS = ['/api/', '/careers/', '/store/', '/x/'];
+
+const docsSpaceRoot = path.join(__dirname, '../');
+
+const buffer = [];
+
+function write(text) {
+  buffer.push(text);
+}
+
+function save() {
+  const fileContents = [...buffer, ''].join('\n');
+  fse.writeFileSync(path.join(docsSpaceRoot, 'broken-links.txt'), fileContents);
```

Comment:
I'm not a fan of the broken link terminology used. It seems to suggest that we are OK with having broken links. Maybe 

```suggestion
  fse.writeFileSync(path.join(docsSpaceRoot, 'ignore-report-broken-links.txt'), fileContents);
```


---

I wonder if we should have a dot at the beginning for the convention of .gitignore, .eslintignore, etc.

```suggestion
  fse.writeFileSync(path.join(docsSpaceRoot, '.broken-links.txt'), fileContents);
```

but some other don't, e.g. `netlify.toml`.

Reply:
I moved to `link-check-errors` to match the name of the script and added a description in top of the file